Hannah-Sten/TeXiFy-IDEA
# Allow quick label change

Fix #1725 

#### Summary of additions and changes

* Immediately start a rename refactoring after introducing a label for a command/environment for which no reasonable label name can be inferred

#### How to test this pull request

* See the included test cases
* Invoke the "Add label" quickfix on the environment/command in the following latex document 

```latex
\begin{document}
    \begin{lstlisting}
    \end{lstlisting}

    \lstinputlisting[lang=test]{test.txt}
\end{document}
```
